pkg/ui/workspaces/cluster-ui/src/store/clusterLocks/clusterLocks.reducer.ts

```diff
@@ -11,6 +11,7 @@
 import { createSlice, PayloadAction } from "@reduxjs/toolkit";
 import { DOMAIN_NAME, noopReducer } from "../utils";
 import { ClusterLocksResponse, SqlApiResponse } from "src/api";
+import "immer";
```

Comment:
Looks like we can revert these changes with an extra line in `tsconfig.json`. According to `yarn why immer` we have two possible copies: 9.0.3 as a direct dependency and 8.0.1 via `@storybook/react -> @storybook/react-dev-utils -> immer`. PNPM's node_modules/ structure is different than yarn's, so we never saw this issue before. In fact, it only repro's with the Bazel build, not with a direct `yarn build:typescript`! This patch against `pkg/ui/workspaces/cluster-ui/tsconfig.json` seems to fix it:

```diff
diff --git a/pkg/ui/workspaces/cluster-ui/tsconfig.json b/pkg/ui/workspaces/cluster-ui/tsconfig.json
index 26628cf453a..89782f2b459 100644
--- a/pkg/ui/workspaces/cluster-ui/tsconfig.json
+++ b/pkg/ui/workspaces/cluster-ui/tsconfig.json
@@ -18,6 +18,9 @@
     "paths": {
       "@cockroachlabs/crdb-protobuf-client": [
         "./node_modules/@cockroachlabs/crdb-protobuf-client"
+      ],
+      "immer": [
+        "./node_modules/immer"
       ]
     }
   },
```